.pylintrc

```diff
@@ -0,0 +1,561 @@
+[MASTER]
```

Comment:
Customise this file to our project. It should only have the paths and errors to ignore.